lib/bundles.cf

```diff
@@ -439,21 +439,29 @@ bundle agent collect_vars(name, tag, flatten)
       "todo_$(name)"    slist => variablesmatching(".*", $(tag));
 
     !flatten::
-      "$(name)" data => parsejson('{}'),            policy => "free";
+      "$(name)"
+        data => parsejson('{}'),
+        policy => "free";
 
       # this iterates!
-      "$(name)" data => mergedata($(name), '{ "$(todo_$(name))": $(todo_$(name)) }'),
+      "$(name)"
+        data => mergedata($(name), '{ "$(todo_$(name))": $(todo_$(name)) }'),
```

Comment:
This is exactly the one I tried to keep on a single line because it's much clearer that way (since we iterate on `$(name)`).